includes/custom_op_lite.h

```diff
@@ -418,6 +418,21 @@ struct Variadic : public TensorBase {
   TensorPtrs tensors_;
 };
 
+#ifdef USE_CUDA
+
+struct CudaContext {
+  void Init(const OrtW::CustomOpApi& api, const OrtKernelContext& ctx) {
+    const auto& ort_api = api.GetOrtApi();
+    ort_api.KernelContext_GetResource(&ctx, 1, 10000, &cuda_stream);
+    if (!cuda_stream) {
+      ORTX_CXX_API_THROW("Failed to fetch cuda stream from context", ORT_RUNTIME_EXCEPTION);
```

Comment:
Given now API is fallible, returning a status is a better now.

Reply:
The ::Init() is to be called during “CreateTuple(...)”, which return a **tuple** to a compute function.
We cannot return a status upwards to notify the failture.